src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp

```diff
@@ -153,14 +154,24 @@ namespace {
                 auto well2 = std::shared_ptr<Well>(new Well( this->getWell(name, handlerContext.currentStep)));
                 auto connections = std::shared_ptr<WellConnections>( new WellConnections( well2->getConnections()));
                 connections->loadCOMPDAT(record, handlerContext.grid, handlerContext.fieldPropsManager);
-
-                if (well2->updateConnections(connections, handlerContext.grid, handlerContext.fieldPropsManager.get_int("PVTNUM")))
-                    this->updateWell(std::move(well2), handlerContext.currentStep);
-
+                if (well2->updateConnections(connections, handlerContext.grid, handlerContext.fieldPropsManager.get_int("PVTNUM"))) {
+                    this->updateWell(well2, handlerContext.currentStep);
```

Comment:
Did you intentionally drop the `std::move(well2)` argument here?

Reply:
No that was not intentional - reinstated now.